src/saveload/saveload.h

```diff
@@ -365,7 +365,6 @@ enum SaveLoadVersion : uint16_t {
 	SLV_TIMETABLE_START_TICKS,              ///< 321  PR#11468 Convert timetable start from a date to ticks.
 	SLV_TIMETABLE_START_TICKS_FIX,          ///< 322  PR#11557 Fix for missing convert timetable start from a date to ticks.
 	SLV_TIMETABLE_TICKS_TYPE,               ///< 323  PR#11435 Convert timetable current order time to ticks.
-	SLV_WATER_REGIONS,                      ///< 324  PR#10543 Water Regions for ship pathfinder.
```

Comment:
I don't removing this is a good idea, it would mean saves which have the water region chunks wouldn't be loadable in future versions.